### ISSUE TITLE
Export of Annex 60 models fails under Python 2.7

### ISSUE THREAD

**Issue by mlauster** [RWTH-EBC/TEASER#272]
No directory with is created, only the top-level directory with the corresponding .mo file

**Comment by PRemmen** [comment]
The problem seems to be in inline for-loops, in logic.simulation.annex we iterate over a list with loop-variable i, and also iteration with i as loop-variable in an inline for-loop.

e.g.
```python
elif i[1] >= 90:
    zone.weightfactor_ow.append(sum([wall.wf_out for wall in walls]))
    [zone.outer_walls_areas.append(i.area) for i in walls]
```

it seems that python >3.4 can handle it, but not python 2.7

In addition we should raise better warning in "project.calc_all_buildings" if a building in the project can't be calculated and then give the user the possibility to get the error warning, e.g.

```python

calc_all_buildings(self, rais_errors=True):
   if raise_errors is True:
       for bldg in reversed(self.buildings):
           bldg.calc_building_parameter(
                    number_of_elements=self._number_of_elements_calc,
                    merge_windows=self._merge_windows_calc,
                    used_library=self._used_library_calc)
    else:
       try:
            for bldg in reversed(self.buildings):
                bldg.calc_building_parameter(
                    number_of_elements=self._number_of_elements_calc,
                    merge_windows=self._merge_windows_calc,
                    used_library=self._used_library_calc)
        except:
             print("Following building can't be calculated:",bldg.name)
             print("raise_errors==True to get python errors"
             self.buildings.remove(bldg)

**Comment by MichaMans** [comment]
solved with #311
